tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkWriterImpl.java

```diff
@@ -300,7 +308,19 @@ private void checkPageSizeAndMayOpenANewPage() {
 
   private void writePageToPageBuffer() {
     try {
-      pageWriter.writePageHeaderAndDataIntoBuff(pageBuffer);
+      int size = pageWriter
```

Comment:
if numberOfPages == 0, then write page0;
else if numberOfPages ==1, then rewrite page0 and write page1,
else write page_i.

so we can reduce copy page1 data

Reply:
great idea!